pkg/oc/bootstrap/docker/up.go

```diff
@@ -118,8 +118,9 @@ var (
 	// internalTemplateLocations are templates that will be registered in an internal namespace
 	// instead of the openshift namespace.
 	internalTemplateLocations = map[string]string{
-		"logging":         "examples/logging/logging-deployer.yaml",
-		"service catalog": "examples/service-catalog/service-catalog.yaml",
+		"logging":                 "examples/logging/logging-deployer.yaml",
+		"service catalog":         "examples/service-catalog/service-catalog.yaml",
+		"template service broker": "examples/templateservicebroker/templateservicebroker-template.yaml",
```

Comment:
what does this do?  Did I need it before and not realize it?

Reply:
it gets the template installed in the openshift namespace automatically and then we instantiate it from there.  you(I guess?) were installing the template in the tsb namespace so you could instantiate it.  I removed the code that was installing the template in the tsb namespace.

Reply:
openshift-infra namespace i should say.  so hidden from normal users.